docs/help.md

```diff
@@ -14,6 +15,55 @@
 * [Avoid Message Conflict](#avoid-message-conflict)
 * [Best performance for logging to `stdout`](#best-performance-for-stdout)
 
+## Test
+Pino provides a test module with some utility functions; see [`pino.test`](/docs/api.md#test) for more details.
+
+```js
+const pino = require('pino')
+
+test('pino instance should log a info message', async () => {
+  const stream = pino.test.sink()
+  const instance = pino(stream)
+
+  instance.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pino.test.once(stream, expected)
+})
+
+// using tap
+test('pino instance should log a info message using a own assert function', async ({ same }) => {
+  const stream = pino.test.sink()
+  const instance = pino(stream)
+
+  instance.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pino.test.once(stream, expected, same)
+})
+
+// using jest
+function is (received, expected, msg) {
+  expect(received).toStrictEqual(expected)
+}
+
+test('pino instance should log a info message using a own assert function on jest', async () => {
+  const stream = pino.test.sink()
+  const instance = pino(stream)
+
+  instance.info('hello world')
+  instance.info('hi world')
+
+  const expected = [
+    { msg: 'hello world', level: 30 },
+    { msg: 'hi world', level: 30 }
+  ]
+  await pino.test.consecutive(stream, expected, is)
+})
+```
+
+If a complete mock of the pino module is needed, a library as `proxyquire` should be used.
```

Comment:
Why would someone need to do this? What situations could lead to it? This sentence by itself seems very out-of-place.

Reply:
It can be helpful when you only want to test how pino is instantiated without testing the log messages.

In the following example, I would mock the pino module and assert that it is instantiated with a dev/prod transport based on the env and to do it a library as `proxyquire` should be used. Am I missing something?
```js
const pino = require('pino')

const { LOG_LEVEL, ENV } = process.env;
let transport;

const devTransport = {
  target: 'pino-pretty',
  options: {
    colorize: true,
  },
};

const prodTransport = {
  target: 'pino-file',
  options: {
    destination: '/var/log/service.log',
  },
};

if (ENV === 'prod') {
  transport = prodTransport
} else {
  transport = devTransport;
}

exports.logger = pino({ transport, level: LOG_LEVEL }).child({ context: 'some-context' });
```

Reply:
I would just check that `exports.logger.level` matches what I expect, but sure, you could use `proxyquire` to do it as well. My point is, this lone sentence doesn't explain why a reader would want to reach for `proxyquire`. And, as such, makes it a frivolous sentence that will only cause confusion.

Reply:
You are right; let me remove it.